src/hooks/balances/resolvers/query/balances.test.tsx

```diff
@@ -0,0 +1,272 @@
+import {
+  balancesByAddressQueryResolverFactory,
+  BalancesByAddressResolverArgs,
+  Entity,
+  objectToArrayWithoutNull,
+  useBalanceQueryResolvers,
+} from './balances';
+
+import { AssetIds, Balance } from '../../../../generated/graphql';
+import { Resolvers, useQuery } from '@apollo/client';
+import { MockedProvider } from '@apollo/client/testing';
+import TestRenderer, { act } from 'react-test-renderer';
+import { gql } from 'graphql.macro';
+import {
+  getMockApiPromise,
+  nativeAssetBalance,
+  nonNativeAssetBalance,
+} from '../../../polkadotJs/tests/mockUsePolkadotJsContext';
+import { ApiPromise } from '@polkadot/api';
+import waitForExpect from 'wait-for-expect';
+import errors from '../../../../errors';
+waitForExpect.defaults.interval = 1;
+
+jest.mock('../../../polkadotJs/usePolkadotJs', () => ({
+  usePolkadotJsContext: () => {
+    // jest executes before imports, that's why require is here
+    const {
+      mockUsePolkadotJsContext,
+    } = require('../../../polkadotJs/tests/mockUsePolkadotJsContext');
+    return mockUsePolkadotJsContext();
+  },
+}));
+
+describe('hooks/balances/resolvers/query/balances', () => {
```

Comment:
I'd rather keep the test case name simple, the full path is provided by the test runner anyways